worker/test/src/RTC/TestRtpPacket.cpp

```diff
@@ -665,6 +667,8 @@ SCENARIO("parse RTP packets", "[parser][rtp]")
 			0x00, 0x00, 0x00, 0x00,
 			0x00, 0x00, 0x00, 0x00,
 			0x00, 0x00, 0x00, 0x00,
+			0x00, 0x00, 0x00, 0x00,
```

Comment:
same here, please define the correct length to the array rather than adding this extra zeroes.